Sprint-2/debug/2.js

```diff
@@ -12,3 +12,6 @@ console.log(`The last digit of 806 is ${getLastDigit(806)}`);
 
 // This program should tell the user the last digit of each number.
 // Explain why getLastDigit is not working properly - correct the problem
+//because in line 3, a const variable was assigned to num with the value of 103, so whenever the getlastgigit function is callin
```

Comment:
If you kept `const num = 103` AND added the `num` argument to `getLastDigit(num)`...

then called `getLastDigit(12) `

What would happen? Write down your prediction first.

Reply:
If I did the same as you explained, I'll still get the same output, but with 1 more unneeded line. I ran and yes got the same output.